queries.sql

```diff
@@ -84,3 +84,76 @@ FROM (
 GROUP BY species;
 
 
+-- Write queries (using JOIN) to answer the following questions:
+-- What animals belong to Melody Pond?
+SELECT id, name
+    FROM animals AS A 
+    INNER JOIN 
+    (SELECT id as i, full_name FROM owners WHERE full_name = 'Melody Pond') AS B 
+    ON A.owner_id = B.i;
+
+-- List of all animals that their type is Pokemon.
+SELECT id, name 
+    FROM animals AS A
+    JOIN
+    (SELECT id as i, name as n FROM species WHERE name = 'Pokemon') AS B
+    ON A.species_id = B.i;
```

Comment:
kindly note that the joins here ought to be LEFT JOIN and not INNER JOIN as seen in your project. Kindly implement these changes so as to have the right queries to be implemented 😄

Reply:
Thank you, @usorfaitheloho ,
I will change it.